ofproto/ofproto-dpif.c

```diff
@@ -115,6 +112,9 @@ struct ofbundle {
 
     /* Status. */
     bool floodable;          /* True if no port has OFPUTIL_PC_NO_FLOOD set. */
+#if defined(P4OVS)
+    uint8_t p4_bridge_id;
```

Comment:
The convention in this data structure is that each member has an end-of-line comment providing additional information or context.

Reply:
Done